### PR TITLE
Add srcset to amp-img example

### PR DESCRIPTION
**Pull request by juliantoledo**
Create an example that shows an amp-img which uses srcset to demonstrate responsive images #39

### CODE REVIEW

**Comment by juliantoledo** [comment]
@sebastianbenz PTAL?

File: src/20_Components/amp-img.html [hunk 1]
```diff
@@ -21,5 +21,10 @@
   -->
   <amp-img src="/img/amp.jpg" width="1080" height="610" layout="responsive" alt="an image"></amp-img>
 
+  <!--
+    Using `srcset` to specify different images for varying viewport widths and pixel densities (change the width of your browser windows to test it).
```
Comment (sebastianbenz) [inline src/20_Components/amp-img.html]: nit: s/Using/Use/

**Comment by sebastianbenz** [comment]
one nit. Good idea with the different image labels. Otherwise LGTM.

**Comment by juliantoledo** [comment]
Great, fixed the nit.

**Pull request merged**